src/browser/reset.js

```diff
@@ -23,7 +23,7 @@ export async function reset(opts) {
 
 	window.scroll(0, 0);
 
-	await sendMouse({ type: 'move', position: [0, 0] });
+	await sendMouse({ type: 'move', position: [0, 0] }).catch(() => {});
```

Comment:
In `manual` mode, tests are run in your existing browser installs, which can't be controlled this way, so simply ignoring the failures here, though a console warning may be good. These work fine in `watch` mode.